### ISSUE TITLE
 How to generate a thumbnail?

### ISSUE THREAD

**Issue by livimonte** [Lepozepo/S3#122]
It's possible to generate a thumb with a different size and send the two images to S3?

Docs: "S3 now uploads directly from the client to Amazon". 

Maybe with canvas?

**Comment by jamgold** [comment]
Yes, it is possible. I wrote a meteor module [cropuploader](https://github.com/jamgold/cropuploader) that does that.

**Comment by livimonte** [comment]
Thanks 👍
